pkg/process/util/network_filter.go

```diff
@@ -44,25 +49,68 @@ func ParseConnectionFilters(filters map[string][]string) (blacklist []*Connectio
 
 		validFilter := true
 		for _, v := range ports {
-			if v == "*" {
+			// reset validFilter for each port
```

Comment:
Why do we need to reset validFilter every time?  Won't we be throwing away the invalidness of previous filters?